src/main/java/ui/MainSetUp.java

```diff
@@ -0,0 +1,121 @@
+package ui;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+
+public class MainSetUp extends Application {
+
+    private Stage primaryStage;
+    private Scene scene;
+    private Parent rootLayout;
+    private AnchorPane listLayout;
+    private MainController controller;
+
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+
+        this.primaryStage = primaryStage;
+        this.primaryStage.setTitle("JFDI");
+
+        setStageTransparent();
+        loadFonts(); // if any
+        initRootLayout();
+        initView();
+
+    }
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    /***************************
+     *** LEVEL 1 Abstraction ***
+     ***************************/
+
+    private void setStageTransparent() {
+        //this.primaryStage.initStyle(StageStyle.TRANSPARENT);
+    }
+
+    private void loadFonts() {
+        // Implement when team decides to specific font types that needs to be loaded
+        // Font font1 = Font.loadFont(Main.class.getResourceAsStream(
+        //                              <filepath e.g."resource/Oxygen regular.ttf">), <size>);
+        // Font font2 = Font.loadFont(Main.class.getResourceAsStream(
+        //                              <filepath e.g."resource/Oxygen regular.ttf">), <size>);
+    }
+
+    private void initRootLayout() throws IOException {
+
+        rootLayout = (BorderPane) FXMLLoader.load(getClass().getResource("/resources/ui/RootLayout.fxml"));
```

Comment:
If you put the `fxml` files under `src/main/resources/ui` instead of `src/main/java/resources/ui/`, you can just use `getResource("/ui/...")` instead of `getResource("/resources/ui/...")` :wink: Gradle sets that up for us.